src/main/java/com/matchus/domains/location/service/LocationService.java

```diff
@@ -0,0 +1,52 @@
+package com.matchus.domains.location.service;
+
+import com.matchus.domains.location.domain.City;
+import com.matchus.domains.location.domain.Ground;
+import com.matchus.domains.location.domain.Location;
+import com.matchus.domains.location.domain.Region;
+import com.matchus.domains.location.exception.CityNotfoundException;
+import com.matchus.domains.location.exception.GroundNotfoundException;
+import com.matchus.domains.location.exception.RegionNotfoundException;
+import com.matchus.domains.location.repository.CityRepository;
+import com.matchus.domains.location.repository.GroundRepository;
+import com.matchus.domains.location.repository.RegionRepository;
+import com.matchus.global.error.ErrorCode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class LocationService {
+
+	private final CityRepository cityRepository;
+	private final RegionRepository regionRepository;
+	private final GroundRepository groundRepository;
+
+	public LocationService(
+		CityRepository cityRepository,
+		RegionRepository regionRepository,
+		GroundRepository groundRepository
+	) {
+		this.cityRepository = cityRepository;
+		this.regionRepository = regionRepository;
+		this.groundRepository = groundRepository;
+	}
+
+	@Transactional
```

Comment:
```suggestion
	@Transactional(readOnly = true)
```

여기는 데이터 변경이 일어나지 않아서 이렇게 수정하셔도 될 것 같아요!

Reply:
굿굿!